### DISCUSSION TITLE
Camera control touch action: how to hide the little hand and pan suggestion animation?

### DISCUSSION THREAD

**Discussion by NeptuneDeluxe** [google/model-viewer#4606] [Q&A] [answered]
Hello there,
first thing: my level is absolute beginner. 
Now about my issue: I am a visual artist and I know basic HTML and CSS. I am building my own website and I need to display some 3D models. Everything works perfectly but I would like to hide the pointing hand icon and the related animation that appears when 
I include the` camera control touch action`. 
My models are auto rotating and the animation+icon collide with the rotation. I would still like for the user to be able to interact with my model but without the visual suggestion (I prefer to add some text underneath it).
I read the documentation but I can't find something related to this.

Is it something doable for someone with my basic knowledge?

![screenshot-127 0 0 1_5500-2023 12 22-18_57_17](https://github.com/google/model-viewer/assets/154624948/98e168c0-8457-459a-b82e-3062b64805a0)

thanks in advance

**Comment by danilokrulj** [comment, marked as answer]
I think you just set interaction-prompt="none"

**Reply by NeptuneDeluxe** [reply]
Thank you so so much!!!
That worked wonderfully 😊
